bin/node/cli/tests/benchmark_block_works.rs

```diff
@@ -39,7 +39,7 @@ async fn benchmark_block_works() {
 		.arg(base_dir.path())
 		.args(["--from", "1", "--to", "1"])
 		.args(["--repeat", "1"])
-		.args(["--execution", "wasm", "--wasm-execution", "compiled"])
+		.args(["--wasm-execution", "compiled"])
```

Comment:
You could remove this line altogether too while you're at it, since we don't have a non-compiled WASM execution anymore anyway.